components/dashboard/src/projects/NewProject.tsx

```diff
@@ -83,6 +86,32 @@ export default function NewProject() {
         })();
     }, [teams]);
 
+    useEffect(() => {
+        if (selectedRepo) {
+            (async () => {
+
+                try {
+                    const guessedConfigStringPromise = getGitpodService().server.guessRepositoryConfiguration(selectedRepo.cloneUrl);
+                    const repoConfigString = await getGitpodService().server.fetchRepositoryConfiguration(selectedRepo.cloneUrl);
+                    if (repoConfigString) {
+                        setSourceOfConfig("repo");
+                    } else {
+                        setSourceOfConfig("db");
+                        setGuessedConfigString(await guessedConfigStringPromise || `tasks:
```

Comment:
@AlexTugarev Please fix the race condition in a follow-up PR when you get a chance 😊

Reply:
Actually, did it here, please review: https://github.com/gitpod-io/gitpod/pull/7127

Reply:
👍🏻